go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/go-redis/redis/v7 v7.0.0-beta.4
 	github.com/go-yaml/yaml v2.1.0+incompatible
 	github.com/google/go-cmp v0.3.0
+	github.com/gotestyourself/gotestyourself v2.2.0+incompatible // indirect
```

Comment:
I did not found where the dependency is coming from, can you please check if `go mod tidy` would drop it?

Reply:
Thanks for double checking - littered from coding. Cleaned it up